### PR TITLE
Module 4 (Selenium) | test case #6

### PR DESCRIPTION
**Pull request by mikeprokopiv**

### CODE REVIEW

File: src/test/java/com/epam/MobileCategoriesList.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    @Test
+    @DisplayName("Verify Categories list is active(expanded) in when web page is in mobile size")
+    public void mobileCategoriesList() {
+        System.setProperty("webdriver.chrome.driver", "chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.get("http://automationpractice.com/index.php");
+
+        Dimension d = new Dimension(480, 620);
+        driver.manage().window().setSize(d);
+        driver.findElement(By.xpath("//header/div/div[@class='container']/div[@class='row']/div[@id='block_top_menu']/div[@class='cat-title']")).click();
```
Comment (andris17) [inline src/test/java/com/epam/MobileCategoriesList.java:22]: the locator is too long, the last bit makes it as unique as the whole, the last two bits makes it fairly stable. why did you need all the rest?

File: src/test/java/com/epam/MobileCategoriesList.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+        //to verify is the list is active we are checking style attribute. If it has value block == active if none == non active
+        driver.findElement(By.xpath("//header/div/div[@class='container']/div[@class='row']/div[@id='block_top_menu']/ul[contains(@style, 'display: block;')]"));
```
Comment (andris17) [inline src/test/java/com/epam/MobileCategoriesList.java:25]: same as above, it's overcomplicated without tan obvious reason. the id is a great help, and the ul's style parameter seems not to add anything. did I miss the point in adding the rest?

**Review by andris17 — APPROVED** [review]

File: src/test/java/com/epam/MobileCategoriesList.java (hunk 1 above)
Comment (mikeprokopiv) [reply src/test/java/com/epam/MobileCategoriesList.java:22]: agree, but unfortunately due to the site is not longer available I can not verify the fix so will leave it as it is so far.

File: src/test/java/com/epam/MobileCategoriesList.java (hunk 2 above)
Comment (mikeprokopiv) [reply src/test/java/com/epam/MobileCategoriesList.java:25]: agree, but unfortunately due to the site being no longer available I can not verify the fix so will leave it as it is so far.
